about.html

```diff
@@ -12,8 +12,25 @@
 </head>
 
 <body>
+
+  <nav>
+    <img class="jumbo-index" src="img/lambda-black.png" alt="Lambda logo in black">
+
+        <a class="home" href="/index.html">Home</a>
+        <a class="about" href="/about.html">About</a>
+        <a class="products" href="#products">Products</a>
+        <a class ="blog" href="#blog">Blog</a>
+        <a class ="contact" href="#contact">Contact</a>
+</nav>
+  
+
     <div class="container about-page">
      
+      <section class="img2">
+        <img src="img/jumbo-about.png" alt="Skyscraper views">
+    </section>
+
+
```

Comment:
For each of your sections in the middle of the page, you can wrap them in a div or a semantic element and target them with flexbox.